php/libraries/BVL_Feedback_Panel.class.inc

```diff
@@ -33,9 +33,16 @@ class BVL_Feedback_Panel
     /**
      * The feedback thread that this panel is for
      *
-     * @var NDB_BVL_Feedback
+     * @private NDB_BVL_Feedback
      */
-    var $_feedbackThread;
+    protected $feedbackThread;
```

Comment:
The doc (`@private`) and the code (`protected`) here still don't seem to agree.

Reply:
Thanks, just fixed it above